test/distributed/test_c10d.py

```diff
@@ -340,6 +340,82 @@ def _test_numkeys_delkeys(self, fs):
         self.assertEqual(b"value1", fs.get("key1"))
         self.assertEqual(b"value2", fs.get("key4"))
 
+        store = self._create_store()
+        keys = [str(i) for i in range(10)]
+        t0 = time.perf_counter()
+        [store.set(k, k) for k in keys]
+        dur_set = time.perf_counter()
+        t0 = time.perf_counter()
+        [store.delete_key(k) for k in keys]
+        dur_delete = time.perf_counter()
+        print(dur_set, dur_delete)
```

Comment:
I would keep this part if you are able to add an assert when comparing the duration and it is consistent. I don't think we need the tests below

Reply:
Oh, sorry, I didn't mean to have that in there. This part doesn't actually check anything useful related to the fix.

Reply:
looks like this wasn't updated